dartagnan/src/main/antlr4/LitmusPTX.g4

```diff
@@ -108,13 +108,33 @@ storeRegister
 
 loadInstruction
     :   localConstant
+    |   localAdd
+    |   localSub
+    |   localMul
+    |   localDiv
     |   loadLocation
     ;
 
 localConstant
     :   load Period mo (Period scope)? register Comma constant
     ;
 
+localAdd
+    :   Add register Comma register Comma constant
+    ;
+
+localSub
+    :   Sub register Comma register Comma constant
+    ;
+
+localMul
+    :   Mul register Comma register Comma constant
```

Comment:
Should it be `Mult`?

Reply:
The instruction is called `mul` in PTX ISA. Also, the only other grammar we have with a similar rule (LLVM) also uses `Mul`

Reply:
Okay, but I think the [token](https://github.com/hernanponcedeleon/Dat3M/blob/ptx-examples/dartagnan/src/main/antlr4/LitmusPTX.g4#L336) used here was named as `Mult` in this file?

Reply:
There is definitely a mismatch between our syntax and the PTX one (not only on mult, but also plus and minus). The alloy tests seems to use the correct `add, dec ...` syntax. 

@tonghaining do you remember why you opted for using `plus, minus, mult ...` in our grammar?

Reply:
I updated the grammar and the tests

Reply:
> There is definitely a mismatch between our syntax and the PTX one (not only on mult, but also plus and minus). The alloy tests seems to use the correct `add, dec ...` syntax.
> 
> @tonghaining do you remember why you opted for using `plus, minus, mult ...` in our grammar?

No I don't remember, I also think we should update them